packages/shared-typedoc/package.json

```
@@ -0,0 +1,19 @@
+{
+  "private": true,
+  "name": "@coveo/shared-typedoc",
+  "type": "module",
+  "main": "dist/index.js",
+  "files": [
+    "dist",
+    "configs",
+    "tsconfig.json"
+  ],
+  "scripts": {
+    "build": "npx tsc -p tsconfig.json"
```

After adding the `project.json` file, you can change this line to this.
```suggestion
    "build": "nx build"
```